setup.py

```diff
@@ -301,7 +301,7 @@ def run_tests(self):
             "doc8",
             "pyenchant",
             "readme_renderer >= 16.0",
-            "sphinx != 1.6.1",
+            "sphinx != 1.6.1, != 1.6.2",
```

Comment:
Is there progress in resolving our issue upstream? And, if not, should we just do `sphinx < 1.6.1` for now?